Build/CMakeLists.txt

```diff
@@ -173,6 +183,14 @@ else()
 		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")
 	endif()
 
+	# Set compiler flag for disabling RTTI
+	set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
+
+	# Disable exception-handling
+	if (NOT CPP_EXCEPTIONS_ENABLED)
+		set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-exceptions")
```

Comment:
It turns out that my Android builds work just fine after all, since I was already disabling exceptions everywhere with the help of `set(CMAKE_ANDROID_EXCEPTIONS FALSE)` in my toolchain file. Although, with that variable being introduced in CMake 3.20 it might just be that the recent CMake version bump in Jolt helped me here.

But, this does result in `-fno-exceptions` being added twice, once from the Android NDK toolchain file and once from this line here. This is fine obviously, and frankly I'm not sure how you'd distinguish between a cross-compilation using the NDK versus the Gradle stuff you're doing anyway. Either way, with this new `CPP_EXCEPTIONS_ENABLED` option it seems unlikely that you'd find yourself with some warning about conflicting/overriding flags that couldn't be resolved by just flipping that option, assuming Clang even emits such a warning in the first place.

Reply:
Cool. I'm doing a final performance comparison and then I'll merge the change.